PVB Linx Groep 6/Assets/Code/Scripts/UI/Minimap.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Minimap : MonoBehaviour
+{
+    [SerializeField] private Camera _minimapCam;
+
+    [SerializeField] private float _maxZoomSize;
+    [SerializeField] private float _minZoomSize;
+    
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Alpha9) && _minimapCam.orthographicSize <= _maxZoomSize)
+            _minimapCam.orthographicSize += 0.25f;
```

Comment:
Don't hard code it, I would actually just create an value that you can change in the editor instead of actually hardcoding it, other then that, the code is quite solid